doc/api/v8.md

```diff
@@ -390,6 +390,82 @@ The API is a no-op if `--heapsnapshot-near-heap-limit` is already set from the
 command line or the API is called more than once. `limit` must be a positive
 integer. See [`--heapsnapshot-near-heap-limit`][] for more information.
 
+## `v8.collectGCProfile(options)`
+
+<!-- YAML
+added: REPLACEME
+-->
+
+* `options` {Object}
+  * `duration` {number} how long you want to collect the data.
+
+* Returns: {Promise}
+
+This API collects GC data over a period of time and returns an object when
+the `Promise` resolves. The content is as follows.
+
+```json
+{
```

Comment:
Do we determine this format or does v8? If it's v8, mentioning here that the format is a v8-specific implementation detail would be good.

Reply:
we determine this format, like `process.report` do.